# For Timestamp and Seq

Header header


# The readings of rpm of both wheels, and their rates of change

int32 rpm_1
int32 rpm_2


# The encoder values coming from both wheels

int32 encoder_1
int32 encoder_2


# MaxRPM

int16 mxrpm
